roles/ci_dcn_site/templates/service-values.yaml.j2

```diff
@@ -94,7 +94,7 @@ data:
 {% endif %}
 {% endfor %}
   manila:
-    enabled: false
+    enabled: true
     manilaAPI:
       customServiceConfig: |
         [DEFAULT]
```

Comment:
In DCN, according to https://github.com/openstack-k8s-operators/architecture/tree/main/examples/dt/dcn, there are three Ceph clusters.. 

You'd need the appropriate Ceph configuration plugged to a specific instance of "manila-share"; and a corresponding extra-mount with the ceph secret. 

See how cinder volume is configured beginning line 27 in this file.. 

We need a change to the "architecture" repo that enables setting "ManilaShares" similar to [CinderVolumes](https://github.com/openstack-k8s-operators/architecture/blob/b945fba2e12df8b183125cfb2c5b8db699887d42/dt/dcn/edpm-post-ceph/nodeset/kustomization.yaml#L129) so we can do that..

Reply:
Correct, and also, we might need to change `manilaShares` here to iterate over the AZs:

```yaml
manilaShares:
{% for _ceph in _ceph_vars_list %}
    {{ _ceph.cifmw_ceph_client_cluster }}:
      customServiceConfig: |
          [DEFAULT]
          enabled_share_backends = cephfs
          enabled_share_protocols = cephfs
          [cephfs]
          driver_handles_share_servers = False
          share_backend_name = cephfs
          share_driver = manila.share.drivers.cephfs.driver.CephFSDriver
          cephfs_conf_path = /etc/ceph/{{ _ceph.cifmw_ceph_client_cluster }}.conf
          cephfs_cluster_name = {{ _ceph.cifmw_ceph_client_cluster }}
          cephfs_auth_id=openstack
          cephfs_volume_mode = 0755
          cephfs_protocol_helper_type = CEPHFS
{% endfor %}
```